### PR TITLE
Fix for DeX mouse offset on Android 13

### PR DESCRIPTION
**Pull request by MaelImhof**
Hi,

**TL;DR :** This is a fix for #301.

## Problem

When used in Samsung DeX mode, the bVNC and aRDP apps have an offset between the Samsung DeX cursor and the actual remote cursor (of the viewed machine). I could also reproduce the issue on Opaque.

image

For some reason, I was only able to reproduce the issue on Android 13. Android 14 seems to be fine.

## Change Log

- Implemented `getTitleBarHeight` in `bVNC/src/main/java/com/iiordanov/bVNC/input/TouchInputHandlerGeneric.java` (as suggested in the issue, the offset seems to be the height of the window's toolbar)
- Updated `getY` in the same file to include the scaled height of the title bar
- Updated how the `y` coordinate is computed in `handleMouseActions` in the same file to include the scaled height of the title bar

## Behavior

- Fixes the issue on Android 13 *(tested)*
- Does not cause any issue on Android 14 *(tested)*
- Should work on older devices as well *(expected, I do not have an older device but the conditions are in place in the code)*

Please let me know if anything is unclear or if you would like something changed in the PR.

Cheers !

### CODE REVIEW

**Comment by iiordanov** [comment]
I took care of this more efficiently with this commit.

https://github.com/iiordanov/remote-desktop-clients/commit/f75e458e37d5db81ac7d4e6117c8c7e2be63402a

The fix is out in 6.1.7 in Beta on Google Play and in the releases

https://github.com/iiordanov/remote-desktop-clients/releases/tag/RELEASE-v6.1.7
